src/getting-started/toolchain.md

```diff
@@ -0,0 +1,124 @@
+## Rust Toolchain
+
+### Rust Installation
+
+Make sure you have [Rust][rust-lang-org] installed. If not, see the instructions on the [rustup][rustup.rs-website] website.
+
+> 🚨 **Warning**: When using Unix based systems, installing Rust via a system package manager (e.g. `brew`, `apt`, `dnf`, etc.) can result in various issues and incompatibilities, so it's best to use [rustup][rustup.rs-website] instead.
```

Comment:
Is there any official documentation going into detail about this? Might be nice to include a link justifying the statement if we're able to.

Reply:
The closest thing I've found is https://rust-lang.github.io/rustup/installation/other.html#using-a-package-manager